src/discordChannelMonitor/index.js

```diff
@@ -0,0 +1,22 @@
+const ChannelMonitor = require('./DiscordChannelMonitor.js');
+
+const channelMonitors = [
```

Comment:
Could this be a map (channel id -> handler)? This way we don't have to iterate the array every time we want to handle a message. Also, we don't have to call every monitor instance, but only the one matching the channel?